### ISSUE TITLE
About the loss function

### ISSUE THREAD

**Issue by Yuzuriha-Inori-x** [WanJJJh/DiffAns#2]
Hi, I'm having some trouble with the code. The KL divergence loss is used in the article, but the CE loss is used in the code you give. Can you tell me what this situation is about?

**Comment by WanJJJh** [comment]
Sorry about that confusion. 
We do like this following the reference work  Diffusionret: Generative text-video retrieval with diffusion
model. This also uses the CE loss in code while KL loss is reported in paper.
The equation of KL is D_{KL}(p||q)=\Sigma p(log p-log q).
Actually we use label as p and logit as q. The item p log p is 0 while p is 1. So that above equation is equal to CE, which is -\Sigma p log q

**Comment by Yuzuriha-Inori-x** [comment]
Thank you very much for your reply.
